libcxx/include/module.modulemap

```diff
@@ -2008,6 +2008,7 @@ module std_private_type_traits_is_void                                   [system
   export std_private_type_traits_integral_constant
 }
 module std_private_type_traits_is_volatile                               [system] { header "__type_traits/is_volatile.h" }
+module std_private_type_traits_is_within_lifetime                        [system] { header "__type_traits/is_within_lifetime.h" }
```

Comment:
When you rebase onto `main`, I would recommend you discard the changes to the `modulemap` entirely and then re-add them again. It's going to be less confusing than resolving the merge conflict.